src/candid_rpc/cketh_conversion.rs

```diff
@@ -92,6 +92,41 @@ pub(super) fn from_fee_history(
     }
 }
 
+pub(super) fn from_transaction_receipt(
+    value: cketh_common::eth_rpc_client::responses::TransactionReceipt,
+) -> evm_rpc_types::TransactionReceipt {
+    evm_rpc_types::TransactionReceipt {
+        block_hash: Hex32::from(value.block_hash.0),
+        block_number: from_checked_amount_of(value.block_number),
+        effective_gas_price: from_checked_amount_of(value.effective_gas_price),
+        gas_used: from_checked_amount_of(value.gas_used),
+        status: match value.status {
+            cketh_common::eth_rpc_client::responses::TransactionStatus::Success => {
+                Nat256::from(1_u8)
+            }
+            cketh_common::eth_rpc_client::responses::TransactionStatus::Failure => {
+                Nat256::from(0_u8)
+            }
+        },
+        transaction_hash: Hex32::from(value.transaction_hash.0),
+        // TODO 243: responses types from querying JSON-RPC providers should be strongly typed
+        // for all the following fields: contract_address, from, logs_bloom, to, transaction_index, tx_type
```

Comment:
Various fields were added to the initial `TransactionReceipt` in the `evm-rpc-branch`; however, they are not strongly typed, which I think should be improved (e.g., see comments about `logs_bloom` below).